engine/schema/src/com/cloud/storage/VolumeDetailVO.java

```diff
@@ -80,4 +80,23 @@ public boolean isDisplay() {
         return display;
     }
 
+    public void setId(long id) {
```

Comment:
Why these setters are needed? There is already a ctor which takes all the parameters.